### PR TITLE
Add named params & outer origin to diff operation

### LINKED ISSUE #19: Empty test query in PR from community same branch
We had a PR from community https://github.com/ckeditor/ckeditor4/pull/4837 which was set up on the same branch name as the target branch but from the fork.

It ends with an empty test query, so we have no test run: https://app.travis-ci.com/github/ckeditor/ckeditor4/jobs/532205765

We should be able to detect differences in another origin.

### PR DESCRIPTION
**Pull request by sculpt0r**
Add origin to the ckeditor4 repository which comes from CLI. Then diff target branch with a branch from that origin. Only if any slug for an external repository is passed and it can't be our ckeditor4 main repo slug.

So, the extra origin should be added only for PR's that come from other forks. This PR also requires changes in the travis.yml on the main repo. ~I'm going to prepare them.~ Please, take a look at: https://github.com/ckeditor/ckeditor4/pull/4972

Closes #19

### CODE REVIEW

**Comment by sculpt0r** [comment]
I added an argument parser, so we can use naming parameters. Since there is a lot of arguments - it will be easier to maintain them from the command line and inside the code. Since they are mostly dependent on the CI - it is hard to hold them inside the configuration `json` file.

**Review by jacekbogdanski — CHANGES_REQUESTED** [review]
I have a feeling that we cannot introduce such breaking change even if we can't come up quickly with anyone who would use this repository. We can assume that there are developers which are integrating CKEditor 4 CI with their own tooling, so it's risky to just break the whole bender runner command line. Also, it's hard to test these changes right now.

I propose to:
* create `development` branch for development as we are already using `master` in "production". Let's freeze `master` for some time (e.g. 2 releases of CKE4)
* start versioning bender runner (I've already created v0.1.0 for `master`)
* use versioned version for CKE4 instead of `master` branch in https://github.com/ckeditor/ckeditor4/pull/4972 so we can see if it's working properly
* create PR in CKE4 adressing the issue in #19, so we can see everything works as expected
* publish bender runner on NPM to simplify the whole process (we may skip this part for now)

**Comment by sculpt0r** [comment]
Ready for review:
- added `.editorconfig` since trimming the last empty line by VSC was a little bit frustrating
- added logs with exit codes - because the spawned process may exit properly, but with error code and data outputted on standard error output
- Bumped package version to `1.0.0` since we are introducing breaking change with naming parameters
- yargs parser was downgraded to be compatible with Node 10

https://github.com/ckeditor/ckeditor4/pull/4972 PR contains changes that should be synced with this PR. The Travis cfg temporarily uses this repo on this branch (until we create the first release). Also, there is a dummy change in one plugin to verify if diff query is generating properly. This change should be reverted before the actual merge.

File: package.json [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"start": "node src/runner.js",
+		"sub:bender": "node src/bender.js",
+		"sub:server": "node src/server.js",
+		"test": "mocha"
+	},
+	"author": "",
+	"license": "ISC",
```
Comment (jacekbogdanski) [inline package.json]: We should use the same license and author as for other public repositories, see https://github.com/ckeditor/ckeditor4-integrations-common/blob/master/package.json#L23
We should also include the license file.

**Review by jacekbogdanski — APPROVED** [review]
Looks good! I've downgraded the package as versioning should happen during tagging.

**Pull request merged**